### DISCUSSION TITLE
김민수 - 교통/공공시설 - 서울시 공공자전거 따릉이 이용의 주요 요소 파악 및 수요 예측

### DISCUSSION THREAD

**Discussion by Minsoo0405** [dinnercoding/wanted-pre-onboarding-challenge-data#107] [2023년 11월 프리온보딩 데이터 챌린지]
# 도메인 선정 이유
- 서울 길거리를 다니다보면 어떤 곳은 따릉이가 많고, 어떤 곳은 거치대가 비여있는 곳을 보면서 이용자가 지역에 따라 이용률이 다를 수 있겠다는 생각이 들었습니다.
- 최근 기사를 통해 따릉이의 수가 증가하면서 운영적자가 약 100억에 이르렀다는 것을 보게 되어 이용에 미치는 주요 요소를 알고 싶어 선택하게 되었습니다.

# 구체적인 문제 정의
- 따릉이 이용에 미치는 주요 요소가 어떤 것이 있는지 파악합니다.
- 해당 요소들을 활용하여 따릉이의 월별 수요 예측을 하고자 합니다.
- 추가로 각 지역의 공공자전거 관리소에서 따릉이 관리(수리, 점검 등)를 위한 최적의 경로를 알고 싶습니다.

# 데이터 업로드
- 참고 기사
  - TBS뉴스 <서울 시민의 발 '따릉이' 어떻게 관리되고 있을까?> 
     https://tbs.seoul.kr/news/newsView.do?typ_800=7&idx_800=3507701&seq_800=20500380
  - 이투데이 <[단독] 따릉이 연체료 연간 10억…1인 최고 납부액 118만원> 
     https://www.etoday.co.kr/news/view/2288511
- 데이터 
  - 자전거 서울 공유(케글): https://www.kaggle.com/datasets/willianoliveiragibin/bike-seoul-sharing/data
  - 서울시 공공자전거 이용현황(서울 열린데이터): https://data.seoul.go.kr/dataList/OA-14994/F/1/datasetView.do
- 추가 필요 데이터
  - 공공자전거 대여소명, 대여소 주소, 대여소 위도 및 경도
  - 공공자전거 관리소명, 관리소 주소, 관리소 위도 및 경도

**Comment by ghost** [comment]
민수님! 저도 따릉이를 자주 이용하다보니 민수님께서 궁금하셨던 부분 저도 궁금했었는데요~
따릉이 이용에 미치는 요인들을 분석하면 운영을 어떻게 효율적으로 할 수 있을지 등에 대해서 분석해보시면 재미있을 것 같아요!
기대해보겠습니다 ㅎㅎ
